$NetBSD: patch-configure,v 1.8 2019/02/12 20:47:14 adam Exp $

Recognize sparc and powerpc linux (or BSD...) platforms.
Don't do doxygen.
Remove bogus diff check.

--- configure.orig	2019-02-04 17:02:33.000000000 +0000
+++ configure
@@ -114,6 +114,8 @@ all_platforms="${all_platforms} armv7-wi
 all_platforms="${all_platforms} armv7s-darwin-gcc"
 all_platforms="${all_platforms} armv8-linux-gcc"
 all_platforms="${all_platforms} mips32-linux-gcc"
+all_platforms="${all_platforms} powerpc-linux-gcc"
+all_platforms="${all_platforms} sparc-linux-gcc"
 all_platforms="${all_platforms} mips64-linux-gcc"
 all_platforms="${all_platforms} ppc64le-linux-gcc"
 all_platforms="${all_platforms} sparc-solaris-gcc"
@@ -166,9 +168,6 @@ for t in ${all_targets}; do
     [ -f "${source_path}/${t}.mk" ] && enable_feature ${t}
 done
 
-if ! diff --version >/dev/null; then
-  die "diff missing: Try installing diffutils via your package manager."
-fi
 
 if ! perl --version >/dev/null; then
     die "Perl is required to build"
@@ -181,19 +180,6 @@ if [ "`cd \"${source_path}\" && pwd`" !=
   fi
 fi
 
-# check installed doxygen version
-doxy_version=$(doxygen --version 2>/dev/null)
-doxy_major=${doxy_version%%.*}
-if [ ${doxy_major:-0} -ge 1 ]; then
-    doxy_version=${doxy_version#*.}
-    doxy_minor=${doxy_version%%.*}
-    doxy_patch=${doxy_version##*.}
-
-    [ $doxy_major -gt 1 ] && enable_feature doxygen
-    [ $doxy_minor -gt 5 ] && enable_feature doxygen
-    [ $doxy_minor -eq 5 ] && [ $doxy_patch -ge 3 ] && enable_feature doxygen
-fi
-
 # disable codecs when their source directory does not exist
 [ -d "${source_path}/vp8" ] || disable_codec vp8
 [ -d "${source_path}/vp9" ] || disable_codec vp9
